FRAMEWORKS.yml

```
@@ -133,7 +133,7 @@ python:
     website: hug.rest
   bocadillo:
     language: "3.7"
-    version: "0.11"
+    version: "0.12"
```

v0.12 is not out yet! :)

```suggestion
    version: "0.11.2"
```

---

wrong commit ;-)

II prefer using X.Y format

thanks for catch up